<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1347</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>311</width>
      <height>156</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_8">
       <property name="text">
        <string>ШЕСТИУГОЛЬНИК</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_7">
       <property name="text">
        <string>ПРЯМОУГОЛЬНИК</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>КВАДРАТ</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>РОМБ</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>КРУГ</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>ТРЕУГОЛЬНИК</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>ЗВЕЗДА (5)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>ЗВЕЗДА (6)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_9">
       <property name="text">
        <string>ЗВЕЗДА (8)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushButton_10">
       <property name="text">
        <string>ВОСЬМИУГОЛЬНИК</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>170</y>
      <width>311</width>
      <height>532</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="10" column="0">
      <widget class="QPushButton" name="pushButton_11">
       <property name="text">
        <string>&lt;-</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QPushButton" name="pushButton_12">
       <property name="text">
        <string>-&gt;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>           Y:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Положение:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Обновить Х:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Обновить У:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Периметр:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Размер:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Площадь:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2" rowspan="2">
      <widget class="QSlider" name="horizontalSlider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>           X:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QTextBrowser" name="textBrowser_3"/>
     </item>
     <item row="1" column="2">
      <widget class="QTextBrowser" name="textBrowser">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QTextBrowser" name="textBrowser_4"/>
     </item>
     <item row="13" column="0">
      <widget class="QTextEdit" name="textEdit"/>
     </item>
     <item row="13" column="2">
      <widget class="QTextEdit" name="textEdit_2"/>
     </item>
     <item row="9" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_14">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_15">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="2">
      <widget class="QTextBrowser" name="textBrowser_2">
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>641</width>
      <height>681</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>710</y>
      <width>87</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Обновить</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_3">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>10</y>
      <width>79</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_4">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>60</y>
      <width>79</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_5">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>110</y>
      <width>79</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_6">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>140</y>
      <width>79</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>10</y>
      <width>254</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить радиус круга:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_16">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>30</y>
      <width>254</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>обновить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>60</y>
      <width>254</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить сторону квадрата:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_17">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>80</y>
      <width>254</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>обновить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>100</y>
      <width>254</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить диагонали ромба:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_18">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>130</y>
      <width>254</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>обновить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>170</y>
      <width>252</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить стороны прямоугольника:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_19">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>200</y>
      <width>254</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>обновить</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_7">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>180</y>
      <width>79</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_8">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>210</y>
      <width>79</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>230</y>
      <width>254</width>
      <height>47</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить сторону шестиугольника:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_20">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>260</y>
      <width>254</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>обновить</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_9">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>260</y>
      <width>79</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>320</y>
      <width>254</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Внешний и внутренний радиус:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_22">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>370</y>
      <width>254</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>обновить</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_11">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>340</y>
      <width>77</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_12">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>370</y>
      <width>77</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>370</y>
      <width>254</width>
      <height>110</height>
     </rect>
    </property>
    <property name="text">
     <string>Внешний и внутренний радиус:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_23">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>440</y>
      <width>254</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>обновить</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_13">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>420</y>
      <width>77</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_14">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>450</y>
      <width>77</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>470</y>
      <width>254</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Внешний и внутренний радиус:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_24">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>510</y>
      <width>254</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>обновить</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_15">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>490</y>
      <width>77</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_16">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>520</y>
      <width>77</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>170</y>
      <width>254</width>
      <height>259</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить сторону треугольника:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_25">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>310</y>
      <width>254</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>обновить</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_17">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>300</y>
      <width>77</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>570</y>
      <width>271</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить сторону восьмиугольника:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_21">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>590</y>
      <width>251</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>обновить</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_10">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>570</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1347</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
